### PR TITLE
Text edits for Variables -> Parameters

### PR DESCRIPTION
**Pull request by andrew-lastmile**
Our documentation and editor seems to go back and forth between variables and parameters, sometimes both. Standardizing on one: `parameters` for now, but we can swap to `variables`.

### CODE REVIEW

File: python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default memo(function GlobalParametersContainer({
       >
         <Accordion.Item value="parameters">
           <Accordion.Control>
-            <Text color="blue">Global Variables (Parameters) {"{}"}</Text>
+            <Text color="blue">Global Parameters</Text>
```
Comment (rholinshead) [inline python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx:33]: Do we want to keep the {"{}"} here?

**Review by rholinshead — APPROVED** [review]
LGTM, just a question

File: python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx (hunk 1 above)
Comment (andrew-lastmile) [reply python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx:33]: hmmm, good question. We can keep it for now. Let me add that change.

**Review by rholinshead — APPROVED** [review]

**Review by rossdanlm — APPROVED** [review]
I like it!

**Pull request merged**